tests/integration/test_cloudformation.py

```diff
@@ -113,6 +113,30 @@ def handler(event, context):
     }
 })
 
+TEST_CHANGE_SET_BODY = """
+Parameters:
+  EnvironmentType:
+    Type: String
+    Default: local
+    AllowedValues:
+      - prod
+      - stage
+      - dev
+      - local
+
+Conditions:
+  IsProd:
+    !Equals [ !Ref EnvironmentType, prod ]
+
+Resources:
+  MyBaseDomainName:
+    Type: AWS::SSM::Parameter
```

Comment:
If I'm not mistaken, the `AWS::SSM::Parameter` resource type is not yet supported, that is, upon executing the change set, the SSM parameter will not actually get created. But we can tackle that in a follow-up PR. 👍

Reply:
obsolete - added in #2197